python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -72,16 +159,16 @@ def get_grouped_stats(
         df_aux_metrics = df.groupby(query_key_col).apply(
             lambda grp: compute_aux_metrics_on_query_group(
                 query_group=grp,
-                label_col=label_col,
+                label_col=artificial_click_col,
```

Comment:
[Nit] I think `artificial_click` is a loaded term because sometimes the label could just be a binary click/no-click @mohazahran